frontend/src/features/projects/erp/steps/ErpStep.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
```

Comment:
This is actually just a rename of approvalStep->ErpStep

ApprovalStep made sense when the design was to have the ERP and SPL tabs on the same page, but since that is no longer the case updating the name made sense.